extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -222,6 +227,7 @@ export class ResourceTypePickerDialog extends DialogBase {
 					return;
 				}
 				let autoInstallRequired = false;
+				let minVersionCheckFailed = false;
```

Comment:
Add a function/property to toolBase called minimumVersionMet and do all the calculations there.